extensions/database/module/scripts/index/database-import-form.html

```diff
@@ -27,7 +27,7 @@
                                   
                                       <div class="pure-control-group">
                                            <label id="connectionNameLabel" for="name"></label>
-                                           <input bind="connectionNameInput" id="connectionName" type="text" placeholder="" value="" class="pure-input-1-3" required>
+                                           <input spellcheck ="false" bind="connectionNameInput" id="connectionName" type="text" placeholder="" value="" class="pure-input-1-3" required>
```

Comment:
For consistency with the rest of the style of most files, I would use `spellcheck="false"`, not `spellcheck ="false"` (no space before the `=` sign).

Reply:
Thank you for reviewing @wetneb will get back soon with the changes